backend/controller/cronjobs/cronjobs_test.go

```diff
@@ -37,7 +38,7 @@ func TestServiceWithMockDal(t *testing.T) {
 		attemptCountMap: map[string]int{},
 	}
 	conn := sqltest.OpenForTesting(ctx, t)
-	parentDAL, err := db.New(ctx, conn, optional.None[string]())
+	parentDAL, err := db.New(ctx, conn, *encryption.NewBuilder())
```

Comment:
It's not really idiomatic to deref pointers as part of normal code flow, can this and above just return a value instead?

Reply:
Looking down further, the `Builder` should just be a value everywhere, so I think these pointers can be removed.